resqpy/fault.py

```diff
@@ -1614,28 +1614,18 @@ def k_gap_connection_set(grid, skip_inactive = True, feature_name = 'k gap conne
    return kgcs
 
 
-def add_connection_set_and_tmults(model, fault_incl, tmult_dict):
+def add_connection_set_and_tmults(model, fault_incl, tmult_dict = {}):
```

Comment:
We should avoid having mutable default arguments (as per one of the recent Bifrost Python Tip threads).

A common solution is having a default of `None`, and a clause like `if tmult_dict is None: dmult_dict = {}`